prime-router/settings/STLTs/TN/tn-doh.yml

```diff
@@ -0,0 +1,98 @@
+- name: "tn-doh"
+  description: "Tennessee Department of Health"
+  jurisdiction: "STATE"
+  stateCode: "TN"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "tn-doh"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/TN/TN-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: true
+        receivingApplicationName: "tdh-ELR^2.16.840.1.113883.3.773.1.1.3^ISO"
```

Comment:
We should need to do any changes to these settings. They should remain the same from the covid pipeline.
```
  "receivingApplicationName": "tdh-ELR",
  "receivingApplicationOID": "2.16.840.1.113883.3.773.1.1.3",
  "receivingFacilityName": "TDH",
  "receivingFacilityOID": "2.16.840.1.113883.3.773",
```

Reply:
As James explained on another PR, they have to change because they set the batch header fields and those don't have a way to set the .2. and .3 fields separately.